dashboard/app/helpers/curriculum_helper.rb

```diff
@@ -18,7 +18,7 @@ def validate_key_format
       return false
     end
 
-    key_re = /\A#{KEY_CHAR_RE}+\Z/
+    key_re = /\A#{KEY_CHAR_RE}+\Z/o
```

Comment:
For anyone else who (like me) was unaware of this flag, https://www.regular-expressions.info/ruby.html says:

> `/o` causes any `#{…}` substitutions in a particular regex literal to be performed just once, the first time it is evaluated. Otherwise, the substitutions will be performed every time the literal generates a Regexp object.